server/src/server.rs

```diff
@@ -286,28 +286,31 @@ impl ConnectionState {
 pub enum PingConfig {
 	/// The server pings the connected clients continuously at the configured interval but
 	/// doesn't disconnect them if no pongs are received from the client.
-	WithoutInactivityCheck(Duration),
+	OnlyPing(Duration),
```

Comment:
I didn't like this verbose names, any suggestions on this?

I still don't like the naming

Reply:
Hmm, maybe we can join the `OnlyPing` and `Ping` variants into one:
```rs
enum PingConfig {
    Disabled,
    Ping {
        ping_interval: Duration,
        disconnect_after: Option<Duration>
    },
}
```